src/components/one-time-donation/components/FormikStepper.tsx

```diff
@@ -31,7 +31,7 @@ export type GenericFormProps<T> = PropsWithChildren<FormikConfig<T>>
 
 export function FormikStepper<T>({ children, ...props }: GenericFormProps<T>) {
   const childrenArray = React.Children.toArray(children) as React.ReactElement<FormikStepProps>[]
-  const { step, setStep } = useContext(StepsContext)
+  const { step, setStep } = useContext(StepsContext)!
```

Comment:
Why do we need to add the exclamation mark here?

Reply:
I was really searching for a way to get around that. That's why I made the context in the first place, so that I wouldn't have to use the hook and check if the campaign exists everywhere, but couldn't find a way to do it without checks. It is giving out warnings for that too

Reply:
One way out of this situation is to throw an error if the campaign is not found. This will remove the `undefined` from the type chain.

I didn't have time to run the code locally to give better suggestion beside this note...